src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -733,8 +735,7 @@ public void testServiceDiscovery() throws Exception {
         LinkedList<HashMap<String, Object>> metrics = getMetrics();
 
         // 14 = 13 metrics from java.lang + 1 metric explicitly defined in the yaml config file
-        assertEquals(59, metrics.size());
-
+        assertEquals(63, metrics.size());
```

Comment:
do you know why the number of metrics increased by 4 here? (haven't checked in details, but my expectation would be an increase of 2)

(same question on the other test below)

Reply:
Looks like each of the two metrics is submitted with different tags:
```
[type:SimpleTestJavaApp, component, scope:CoolScope, instance:jmx_test_instance, jmx_domain:org.datadog.jmxfetch.test, bean_host:localhost]
[foo:Bar, qux:Baz, instance:jmx_test_instance, jmx_domain:org.datadog.jmxfetch.test]
```

I'm guessing its because of how its registered here:
https://github.com/DataDog/jmxfetch/pull/204/files#diff-974534e55eb97868b68abda77ce93ecbR728
but I'm not too familiar with the tests

Reply:
makes sense, thanks for checking 👍